cluster/juju/layers/kubernetes-master/templates/heapster-controller.yaml

```diff
@@ -0,0 +1,138 @@
+{% set base_metrics_memory = "140Mi" -%}
```

Comment:
This is an exact copy of the upstream template. I didn't tweak any of the variables.